test-data/unit/README.md

```diff
@@ -12,7 +12,6 @@ feature you added. If you added a new `check-*.test` file, it will be autodiscov
 Add the test in this format anywhere in the file:
 
     [case testNewSyntaxBasics]
-    # flags: --python-version 3.6
```

Comment:
It's useful to show an example of how to use custom flags in a test case. Maybe we could change the example to something that tests a new feature in 3.10?